packages/css/src/components/breadcrumb/breadcrumb.scss

```diff
@@ -5,7 +5,7 @@
 
 @import "../../common/text-rendering";
 
-@mixin reset {
+@mixin reset-list {
```

Comment:
Please use the element names. There’s no need for a difference with the type selector for which we reset the user agent styles. That `reset-ul` and `reset-ol` will possibly be the same is okay.